glamox/src/App.tsx

```diff
@@ -1,17 +0,0 @@
-import React from 'react';
-
-import './App.css';
-import { Display } from './Display';
-
-
-
-function App() {
-  
-  return (
-    <div>
```

Comment:
Use React fragments instead of empty <div> tags. Because div tags take longer to load in the browser.